external/llvm-project/mlir/lib/Dialect/GPU/Transforms/SerializeToBlob.cpp

```diff
@@ -71,6 +90,12 @@ void gpu::SerializeToBlobPass::runOnOperation() {
   getOperation()->setAttr(gpuBinaryAnnotation, attr);
 }
 
+LogicalResult
+gpu::SerializeToBlobPass::optimizeLlvm(llvm::Module &llvmModule,
+                                       llvm::TargetMachine &targetMachine) {
+  return success();
```

Comment:
probably need to add comment that in the default implementation in `serializeToBlobPass` no optimization would be carried out.